Makefile.Vx

```diff
@@ -1,3 +1,6 @@
+## THIS FILE IS OUTDATED!!
```

Comment:
It allows to build the module for EPICS 3.13. Which IS outdated but still in use at PSI.